app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -38,6 +38,14 @@ def update
     end
   end
 
+  def reset_password
+    if @user.update_attributes(password: "very_secure_password")
```

Comment:
Interesting. Does it even work?
1. We need to send the reset link to the user. This should be implemented in scope of devise
2. What is the logic of saving password to `very_secure_password`?

Maybe I am missing something.